root/static/scripts/edit/URLCleanup.js

```diff
@@ -1019,6 +1041,64 @@ const CLEANUPS: CleanupEntries = {
             result: /^https:\/\/[^\/]+\.bandcamp\.com\/track\/[\w-]+$/.test(url),
             target: ERROR_TARGETS.ENTITY,
           };
+        case LINK_TYPES.downloadfree.recording:
+        case LINK_TYPES.downloadpurchase.recording:
+        case LINK_TYPES.streamingfree.recording:
+          if (/^(https?:\/\/)?([^\/]+)\.bandcamp\.com\/?$/.test(url)) {
+            return {
+              error: exp.l(
+                `This is a Bandcamp profile, not a page for a specific
+                 recording. Even if it shows a single recording right now,
+                 that can change when the artist releases another.
+                 Please find and add the appropriate recording page
+                 (“{single_url_pattern}”)
```

Comment:
Should it be a recording or a release?
Any actual example at hand?

Reply:
This is shown specifically when trying to add a link to a recording, so the only things allowed should be links like https://pyrrhonband.bandcamp.com/track/the-cost-of-living

Reply:
Can Bandcamp profile points to track?
Any actual example at hand?

Reply:
It can, if just one single is all there is. I've seen it in the past, but not finding any examples right now :)

Reply:
Isn’t single matching release rather than recording?

Also it seems less effort to add the Bandcamp profile URL to the corresponding MB artist first, and then to try to find the correct Bandcamp URL for the current recording/release.

Reply:
Well, if it is still /track, then it belongs at both the release and recording level, really. But if you think the error message is not great, feel free to suggest improvements - this is not a new message, but the same we already had, so any improvements probably should be a separate commit anyway though :)

Reply:
It’s not great but it’s not about this pull request indeed. :)